README.md

```diff
@@ -80,6 +80,26 @@ This project welcomes contributions, suggestions, and feedback. All contribution
 
 For more information on how to contribute Mixed Reality Toolkit for Unity Project, please read [CONTRIBUTING.md](./CONTRIBUTING.md).
 
+## MRTK3 XRI2 to XRI3 migration guide
+
+MRTK3 has been upgraded to use [Unity's XR Interaction Toolkit 3+](https://docs.unity3d.com/Packages/com.unity.xr.interaction.toolkit@3.0/manual/whats-new-3.0.html).  As part of the upgrade several changes were made to properly consume XRI 3 package and adhere to the new patterns.  In a nutshell, the main changes are summarized as follows:
+
+* New controller prefabs and a new rig have been created following the new XRI3 pattern.
+  * The old controllers and rig have been marked as osbolete and renamed as "Obsolete MRTK XR Rig", "Obsolete MRTK LeftHand Controller", "Obsolete MRTK RightHand Controller", "Obsolete MRTK Hand Controller", "Obsolete MRTK Interaction Manager", and "Obsolete MRTK Gaze Controller".
+  * The new controllers and rig retake the original names of the obsolete controllers.
+* New controllers structure have been uniformized so that all of them have the same structure.
```

Comment:
have been modified so that all of them have the same structure.

Reply:
Thx for feedback, implemented in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/839/commits/e14267bbcb572fa95f69535cbc5fc9cbb0a22d00, resolving.